shell/common/shell.h

```diff
@@ -378,8 +378,9 @@ class Shell final : public PlatformView::Delegate,
   std::unique_ptr<ShellIOManager> io_manager_;   // on IO task runner
   std::shared_ptr<fml::SyncSwitch> is_gpu_disabled_sync_switch_;
 
-  fml::WeakPtr<Engine> weak_engine_;          // to be shared across threads
-  fml::WeakPtr<Rasterizer> weak_rasterizer_;  // to be shared across threads
+  fml::WeakPtr<Engine> weak_engine_;  // to be shared across threads
+  fml::TaskRunnerAffineWeakPtr<Rasterizer>
```

Comment:
@iskakaushik Changed after your last approval:
It was implicitly casted from a `TaskRunnerAffineWeakPtr` to a `WeakPtr`